### PR TITLE
Databricks

### PR DESCRIPTION
**Pull request by schottj**
## Description & motivation
Adding details for the new databricks adapter in dbt Cloud. Keeping info about the current connection details, which we'll deprecate alongside how to connect to the new adapter.

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 1]
```diff
@@ -165,6 +165,29 @@ The following fields are available when creating a Databricks connection:
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/databricks-connection.png" title="Configuring a Databricks connection"/>
 
+### New Databricks Connection Details
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
### New Databricks connection details
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 ## Connecting to Databricks
 
-### ODBC
+### Current Databricks Connection Details (to be deprecated June 30, 2022)
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
### Current Databricks connection details (to be deprecated June 30, 2022)
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 ## Connecting to Databricks
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion

You can now connect to Databricks using host and HTTP path of the Databricks cluster or SQL endpoint. For more information, see [New databricks connection details](dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database#new-databricks-connection-details).

```

**Review by runleonarun — DISMISSED** [review]
@schottj Just some minor suggestions! Feel free to merge once they are addressed.

File: website/docs/reference/warehouse-profiles/databricks-profile.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ id: "databricks-profile"
 **Maintained by:** some dbt loving Bricksters
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/databricks-profile.md]:
```suggestion
**Maintained by:** dbt-loving Bricksters  
```

**Review by dataders — COMMENTED** [review]
excited to have this live @schottj ! only thing I noticed was some stale info here. Both the recommended approach and links to doc are stale. I think we nuke the whole sentence.


https://github.com/dbt-labs/docs.getdbt.com/blob/d2e7a0000292d597bb14c8eeebd38dca82a923ce/website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md#L196-L198

**Pull request closed without merging**